data/mods/Magiclysm/Spells/technomancer.json

```
@@ -24,8 +24,8 @@
     "range_increment": 1.0,
     "//": "duration is in moves",
     "min_duration": 6000,
-    "max_duration": 10000,
-    "duration_increment": 400
+    "max_duration": 18000,
+    "duration_increment": 600
```

```suggestion
    "max_duration": 120000,
    "duration_increment": 5700
```
3 minutes is better than before, but still nothing in my eyes. Lets make it 20 minutes at least